src/AppKit/NSAccessibility.cs

```diff
@@ -80,6 +89,93 @@ public static CGPoint GetPointInView (NSView parentView, CGPoint point)
 		{
 			return NSAccessibilityPointInView (parentView, point);
 		}
+
+		[Mac (10,7)]
+		[DllImport (Constants.AppKitLibrary)]
+		static extern void NSAccessibilityPostNotificationWithUserInfo (IntPtr element, IntPtr notification, IntPtr userInfo);
+
+		public static void PostNotification (NSObject element, NSString notification, NSDictionary userInfo)
+		{
+			NSAccessibilityPostNotificationWithUserInfo (element.Handle, notification.Handle, userInfo.Handle);
```

Comment:
Any `null` arguments will cause NRE (common issue in several API here).

If `null` is allowed by the native API then `IntPtr.Zero` must be given to p/invoke.
If not then a null check with a ANE should be thrown.